src/server.js

```diff
@@ -18,6 +18,11 @@ var authFunction = localAuth;
 module.exports = function() {
 	manager = new ClientManager();
 
+	if (!fs.existsSync("client/js/bundle.js")) {
```

Comment:
So, I've tried to move it to `index.js` and I'd definitely keep it where it is: `index.js` doesn't have access to `log` at the moment (probably not a big deal but still) but way more importantly, it's blocking `-h`, the `list` subcommand, etc.

Let's keep it that way for now and we can improve that later.